sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -343,13 +361,14 @@ private async Task TestSingleDrainMode<T>(bool sendToQueue)
                 // Validate that function execution was allowed to complete
                 Assert.True(_drainValidationPostDelay.WaitOne(DrainWaitTimeoutMills + SBTimeoutMills));
                 await jobHost.StopAsync();
+                AssertNoLoggedErrors(host);
             }
         }
 
         private static Action<IHostBuilder> BuildDrainHost<T>()
         {
             return builder =>
-                builder.ConfigureDefaultTestHost<T>(b =>
+                builder.ConfigureWebJobs(b =>
```

Comment:
This was necessary as ConfigureDefaultTestHost added the logging provider, and BuildHost does as well so there were two logging providers and the GetTestLoggerProvider call was failing in the Drain tests.